tests/datadriven/__init__.py

```diff
@@ -52,11 +53,17 @@ class DataDrivenTest(object):
     corresponding to this, and then test that these objects have the
     properties that we expect.
     """
-    def __init__(self, setId, baseDir):
-        self._setId = setId
-        self._dataDir = os.path.join(baseDir, setId)
-        self._gaObject = self.getDataModelClass()(
-            self._setId, self._dataDir)
+    def __init__(self, *args):
+        if len(args) == 0:
```

Comment:
So I can cleanly test the assert methods without doing unnecessary file operations, etc.